packages/stepper/src/stories/Stepper.components.tsx

```diff
@@ -0,0 +1,212 @@
+import { Form, InlineMessageDestructive } from '@talend/design-system';
+import { createContext, useContext, useEffect } from 'react';
+import StepperForm from '../components/StepperForm';
+import { FormProvider, useController, useForm } from 'react-hook-form';
+
+interface BusinessContextApi {
+	setStepsData: (stepData: any) => void;
+	stepsData: any[];
+}
+
+export const BusinessContext = createContext({} as BusinessContextApi);
+
+export const FormComponentStep1 = () => {
+	const { onDisableStep, onEnableStep, onNext, steps, currentStep } = useContext(
```

Comment:
I'm not sure `steps` and `currentStep` should be publicly exposed by `StepperForm.Context`. 
As I think it's required by internal components, maybe you could split your context with internal and external properties?